app/(anon)/steps/3/_components/GeneralPage.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import { generalPageStyles } from './generalPage.style.ts';
```

Comment:
이거 import 이름을 styles로 해주세요
```suggestion
import { styles } from './generalPage.style.ts';
```